src/lang/ResponseLanguage.ts

```diff
@@ -1,22 +1,14 @@
 export const ProductLanguage = [
```

Comment:
would be good to add type or interface for this const

Reply:
I agree with @antoninroam. Also, nit, this should be camelCased not PascalCased